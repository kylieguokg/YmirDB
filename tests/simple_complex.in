set a 7 8 39 1 10 -29
sort a
get a
uniq a
get a
append a 7
get a
sort a
get a
len a
rev a
get a
uniq a
get a
len a
push a 39
append a -29
get a 
sort a
get a
rev a
get a
uniq a
get a
max a
min a
pop a
pluck a 5
get a
max a
min a
bye
