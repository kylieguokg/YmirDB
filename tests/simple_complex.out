> ok

> ok

> [-29 1 7 8 10 39]

> ok

> [-29 1 7 8 10 39]

> ok

> [-29 1 7 8 10 39 7]

> ok

> [-29 1 7 7 8 10 39]

> 7

> ok

> [39 10 8 7 7 1 -29]

> ok

> [39 10 8 7 1 -29]

> 6

> ok

> ok

> [39 39 10 8 7 1 -29 -29]

> ok

> [-29 -29 1 7 8 10 39 39]

> ok

> [39 39 10 8 7 1 -29 -29]

> ok

> [39 10 8 7 1 -29]

> 39

> -29

> 39

> -29

> [10 8 7 1]

> 10

> 1

> bye
